src/components/CheckOutForm.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import styled from 'styled-components';
+import propTypes from 'prop-types';
+
+class CheckOutForm extends React.Component {
+  state = {
+    EmailInput: '',
+  };
+  setEmailInput = EmailInput => {
+    this.setState({ EmailInput });
+  };
+  render() {
+    return (
+      <StyledCheckOutForm>
+        <select>
+          {this.props.rooms.map(room => (
+            <option key={room._id} value={room._id}>
+              {room.name}
+            </option>
+          ))}
+        </select>
+        <input
+          placeholder='email'
+          onChange={event => this.setEmailInput(event.target.value)}
+        />
+        <button>Check Out</button>
+      </StyledCheckOutForm>
+    );
+  }
+}
+CheckOutForm.propTypes = {
+  rooms: propTypes.arrayOf(
+    propTypes.shape({
+      _id: propTypes.string.isRequired,
+      name: propTypes.string.isRequired,
+    }),
+  ).isRequired,
+  hotel_id: propTypes.string.isRequired,
+};
+
+const StyledCheckOutForm = styled.div``;
+
+export default CheckOutForm;
```

Comment:
Do we have two files with `CheckOutForm.js` and `CheckoutForm.js` ? What is the difference there?

Reply:
One of those should be renamed, as the naming is quite confusing.

Reply:
@ConConRob @Pav0l CheckoutForm is for billing but should be reserved for guest checkouts so I'll rename and then Conor can use whatever variation on it he wants

Reply:
I had a CheckOutForm which I renamed to GuestCheckOutForm. This file didn't get removed when I renamed it so I'll make a push that fixed this. I think we should start putting our components in folders based on views.